src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -209,7 +209,7 @@ public int getOstrichPort() {
     }
 
     public String getCloudService() {
-        return getString("cloud.service");
+        return getString("cloud.service", "");
```

Comment:
why change this line?  The original default value is 'S3' which is specified in the properties file.  I think we always want people to explicitly set the properties value in the properties file, not in the java code.